consensus/src/testing/mod.rs

```diff
@@ -86,7 +86,7 @@ pub fn spawn_honest_member(
     let spawner_inner = spawner;
     let unit_loader = Loader::new(units);
     let saved_state = Arc::new(Mutex::new(vec![]));
-    let unit_saver = Saver::new(saved_state.clone());
+    let unit_saver = Saver::from(saved_state.clone());
```

Comment:
Have you tried `.into()` here?

Reply:
Or, even better – if I understand what's happening here correctly then `new` could be returning this `Arc<...>` and you don't have to have the `From` at all. In other places you would just let `(unit_saver, _) = Saver::new()` and here you would have `saved_state` in the second position.

Reply:
Yeah, `(unit_saver, _) = Saver::new()` would work, but I don't know, `::from` looks more intuitive to me.
`into()` is also possible instead. When we use `::from` its easy to track calls in the IDE but I can see that `.into()` is used more often in the repo.